src/_gettsim/shared.py

```diff
@@ -16,43 +16,21 @@ def __repr__(self):
         return str(self)
 
 
-def add_rounding_spec(params_key):
-    """Decorator adding the location of the rounding specification to a function.
-
-    Parameters
-    ----------
-    params_key : str
-        Key of the parameters dictionary where rounding specifications are found. For
-        functions that are not user-written this is just the name of the respective
-        .yaml file.
-
-    Returns
-    -------
-    func : function
-        Function with __info__["rounding_params_key"] attribute
-
-    """
-
-    def inner(func):
-        # Remember data from decorator
-        if not hasattr(func, "__info__"):
-            func.__info__ = {}
-        func.__info__["rounding_params_key"] = params_key
-
-        return func
-
-    return inner
-
-
 TIME_DEPENDENT_FUNCTIONS: dict[str, list[Callable]] = {}
 
 
-def dates_active(
+def policy_info(
+    *,
     start: str = "0001-01-01",
     end: str = "9999-12-31",
     change_name: str | None = None,
```

Comment:
```suggestion
    name_in_dag: str | None = None,
```
I think this makes things clearer, especially given the new context but would have been good right away!